compiler/lexer.nim

```diff
@@ -915,7 +915,7 @@ proc getSymbol(L: var TLexer, tok: var TToken) =
 proc endOperator(L: var TLexer, tok: var TToken, pos: int,
                  hash: Hash) {.inline.} =
   var h = !$hash
-  tok.ident = L.cache.getIdent(addr(L.buf[L.bufpos]), pos - L.bufpos, h)
+  tok.ident = L.cache.getIdent(addr(L.buf[L.bufpos]).toCstring, pos - L.bufpos, h)
```

Comment:
```suggestion
  tok.ident = L.cache.getIdent(L.buf[L.bufpos].addr.toCstring, pos - L.bufpos, h)
```
to be consistent with the line in 904. Also I agree with krux02 here, instead of toCstring you should use casts manually and remove toCstring.